app/src/main/java/io/intrepid/skeleton/screens/example2/Example2Contract.java

```diff
@@ -0,0 +1,15 @@
+package io.intrepid.skeleton.screens.example2;
+
+import io.intrepid.skeleton.base.BasePresenter;
+import io.intrepid.skeleton.base.BaseView;
+
+public class Example2Contract {
+    public interface View extends BaseView {
+
```

Comment:
Yeah, this is a bit tricky. I'm leaning towards View since it's Model **View** Presenter.